app/Mail/ArticleCreated.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Mail;
+
+use App\Models\Article;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class ArticleCreated extends Mailable
+{
+    use Queueable, SerializesModels;
```

Comment:
PSR12: Each individual trait that is imported into a class MUST be included one-per-line and each inclusion MUST have its own use import statement.
https://www.php-fig.org/psr/psr-12/